corehq/ex-submodules/casexml/apps/phone/restore_caching.py

```diff
@@ -24,7 +23,6 @@ def _restore_payload_path_cache_key(domain, user_id, sync_log_id, device_id):
         domain=domain,
         prefix=RESTORE_CACHE_KEY_PREFIX,
         user_id=user_id,
-        version='2.0',
```

Comment:
I'm very pro getting rid of V1. However, I thought I saw [references](https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/phone/tests/test_ota_restore_v3.py) to [V3](https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/case/xml/__init__.py#L7) in the code. Is that not a thing or was that a false start?